test/api_integration/apis/search/sql_search.ts

```diff
@@ -53,7 +56,10 @@ export default function ({ getService }: FtrProviderContext) {
           .expect(200);
         const id = resp1.body.id;
 
-        const resp2 = await supertest.post(`/internal/search/sql/${id}`).send({});
+        const resp2 = await supertest
+          .post(`/internal/search/sql/${id}`)
+          .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+          .send({});
 
         expect(resp2.status).to.be(200);
         expect(resp2.body.id).to.be(id);
```

Comment:
I see that we have set responses to send back `elastic-api-version`. I am not sure how important that is but Is it something worth testing that `request` and `response` have same version number. This comment also applies to other tests.

Reply:
👍 d166ec45ed6